docs/actions/deliver.md

```diff
@@ -406,6 +400,33 @@ The english name of the secondary first sub category you want to set
 The english name of the secondary second sub category you want to set
 </details>
 
+# Submit Build
+
+Select a previously uploaded build and submit it for review.
```

Comment:
To make it easier for search, maybe add something like `Promote an existing build to production` or similar as well as what we already have 👍